### PR TITLE
Qi2016pointnet+slycane9

### PR DESCRIPTION
**Pull request by slycane9**

### CODE REVIEW

File: _summaries/qi2016pointnet_1.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Above we can see that PointNet outperforms other 3D input models, except for MVCNN which has a slight lead.
+
+![segment](qi2016pointnet_1e.png)
+
+In segmentation tests qualitatively seen above, PointNet also achieved State-of-the-art results.  Similar results were achieved in semantic scene segmentation tasks.
+
+![flops](qi2016pointnet_1d.png)
+
+Finally, PointNet achieved much higher classification speeds than other competitive models MVCNN and Subvolume.  The table above shows a 141 times decrease in FLOPS per sample compared to MVCNN.
+
```
Comment (samatharhay) [inline _summaries/qi2016pointnet_1.md:38]: There is a pretty detailed section at the end of the paper that discusses experiments done of the architecture, specifically on order-invariant methods, input and feature transformations, and robustness. I think this is a key section of the paper that should be included in the summary.

File: _summaries/qi2016pointnet_1.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+![arch](qi2016pointnet_1a.png)
+
+Above is the architecture for PointNet.  The first layers go through an input and feature transformer.  These transformers are themselves mini networks and protect against variance in the point cloud input (ensuring we regard different transformations of the point cloud the same way).
+
+
+Another part of the architecture is the split between a classification network and segmentation network.  This allows sharing of global knowledge with local knowledge.  As you can see in the above figure there is a connection between the two networks.
+
+Finally, the MLPs and Max pooling layers are used to simulate a symmetric function.  A symmetric function is needed to protect against input permutations.  Looking at the symmetric function below, the MLPs represent collection h, and max pooling represents g.
+
```
Comment (samatharhay) [inline _summaries/qi2016pointnet_1.md:22]: This section on the implementation needs some more work. I would recommend adding information about the actual input, such as its the unordered nature of it the input data and explain a little more about how the symmetric function works and what its doing.

File: _summaries/qi2016pointnet_1.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+---
+* What is the core idea?
+
+Before this paper, state-of-the-art 3D classification/segmentation models all required 3D input to be transformed into voxel grids or multiple image views.  However, this paper introduces PointNets, which only requires point clouds as input to its model.  Using point clouds as input avoids issues with dense image inputs, and avoid irregularities that are difficult to learn from.
+
+* How is it realized (technically)?
+
+![arch](qi2016pointnet_1a.png)
+
+Above is the architecture for PointNet.  The first layers go through an input and feature transformer.  These transformers are themselves mini networks and protect against variance in the point cloud input (ensuring we regard different transformations of the point cloud the same way).
```
Comment (samatharhay) [inline _summaries/qi2016pointnet_1.md:16]: I would to into more detail about what the mini networks are (also mention that they are the T-nets in the image). I would also discuss the regularization that is done to the feature transformation matrices, and why it is done.

**Review by samatharhay — COMMENTED** [review]
@slycane9 Your summary seems very short and rushed. I noticed that on your technical explanation was somewhat lacking, mostly just pointing to a figure of the architecture and asking the reader to reference it. I would recommend flushing this out a little more, specifically explaining why the authors made the architectural decisions that they did. Also for the results section I think its fine mentioning what the experiments were and then just having a figure with the results, but there is a really good section at the end of the paper that about the architecture analysis that I think you should spend some additional time including, especially about the robustness of the model.

It took me less than a minute to read through your summary, when it should be probably take closer to 3-5 minutes. I would expand it with some more details especially in the technical implementation section.

**Review by zhaoyue-zephyrus — APPROVED** [review]

**Pull request merged**